tasks/sep/base-002-fp-upgrade/SignFromJson.s.sol

```diff
@@ -0,0 +1,366 @@
+    address constant superchainConfigGuardian = 0x7a50f00e8D05b95F98fE38d8BeE366a7324dCf7E;
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "unsafeBlockSigner()(address)"`
+    address constant p2pSequencerAddress = 0xb830b99c95Ea32300039624Cb567d324D4b1D83C;
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "batchInbox()(address)"` and https://docs.base.org/docs/base-contracts/
+    address constant batchInboxAddress = 0xfF00000000000000000000000000000000084532;
+
+    /// @notice Verify against https://docs.base.org/docs/base-contracts/
+    address constant batchSenderAddress = 0x6CDEbe940BC0F26850285cacA097C11c33103E47;
+
+    /// @notice Verify against lib/superchain-registry/superchain/extra/addresses/sepolia/base.json and https://docs.base.org/docs/base-contracts/
+    address constant systemConfigOwner = 0x0fe884546476dDd290eC46318785046ef68a0BA9;
+
+    uint256 constant l2GenesisBlockGasLimit = 45e6;
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "overhead()(uint256)"`
+    uint256 constant gasPriceOracleOverhead = 0;
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "scalar()(bytes32)"`
+    uint256 constant gasPriceOracleScalar = 0x010000000000000000000000000000000000000000000000000a118b0000044d;
+
+    /// @notice Verify with `cast call 0xf272670eb55e895584501d564AfEB048bEd26194 "startBlock()(uint256)"`
+    uint256 constant systemConfigStartBlock = 4370903;
+
+    AddressManager addressManager = AddressManager(0x709c2B8ef4A9feFc629A8a2C1AF424Dc5BD6ad1B);
+
+    address ownerSafe;
+
+    Types.ContractSet proxies;
+
+    /// @notice Sets up the contract
+    function setUp() public {
+        proxies = _getContractSet();
+
+        ownerSafe = vm.envAddress("OWNER_SAFE");
+    }
+
+    function getCodeExceptions() internal pure override returns (address[] memory) {
+        address[] memory shouldHaveCodeExceptions = new address[](4);
+
+        shouldHaveCodeExceptions[0] = systemConfigOwner;
+        shouldHaveCodeExceptions[1] = batchSenderAddress;
+        shouldHaveCodeExceptions[2] = p2pSequencerAddress;
+        shouldHaveCodeExceptions[3] = batchInboxAddress;
+
+        return shouldHaveCodeExceptions;
+    }
+
+    /// @notice Reads the contract addresses from lib/superchain-registry/superchain/extra/addresses/sepolia/base.json
+    function _getContractSet() internal returns (Types.ContractSet memory _proxies) {
+        string memory addressesJson;
+
+        // Read addresses json
+        try vm.readFile(
+            string.concat(vm.projectRoot(), "/lib/superchain-registry/superchain/extra/addresses/sepolia/base.json")
+        ) returns (string memory data) {
+            addressesJson = data;
+        } catch {
+            revert("Failed to read lib/superchain-registry/superchain/extra/addresses/sepolia/base.json");
+        }
```

Comment:
unfortunately this will now need to read toml from the refactored registry.

The path is to the file has changed to `lib/superchain-registry/superchain/configs/sepolia/base.toml` and the read will need to use `yq` with an invocation something like `yq --input-format=toml`. --json`.

Reply:
Refactored to read from `.toml` file